gulpfile.js

```diff
@@ -434,10 +434,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '15.2.1'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.15.1'))
+    .pipe(replace('[ios.current-app-version]', '2.16.1'))
```

Comment:
iOS will probably be 2.16.2:
https://github.com/corona-warn-app/cwa-app-ios/releases/tag/v2.16.0-RC7
```suggestion
    .pipe(replace('[ios.current-app-version]', '2.16.2'))
```

Reply:
@Ein-Tim Thanks.